x-pack/plugins/licensing/server/constants.ts

```diff
@@ -0,0 +1,21 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+export const SERVICE_NAME = 'licensing';
+export const DEFAULT_POLLING_FREQUENCY = 30001; // 30 seconds
```

Comment:
I don't think we should have a dedicated constants file. this value exists only in config context, no one else should read it

Reply:
This is used internally to the plugin, not external consumers per se.